pkg/relayer/cmd/cmd.go

```diff
@@ -0,0 +1,400 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/url"
+
+	"cosmossdk.io/depinject"
+	cosmosclient "github.com/cosmos/cosmos-sdk/client"
+	cosmosflags "github.com/cosmos/cosmos-sdk/client/flags"
+	cosmostx "github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/cmd/signals"
+	"github.com/pokt-network/poktroll/pkg/client/block"
+	eventsquery "github.com/pokt-network/poktroll/pkg/client/events_query"
+	"github.com/pokt-network/poktroll/pkg/client/supplier"
+	"github.com/pokt-network/poktroll/pkg/client/tx"
+	"github.com/pokt-network/poktroll/pkg/relayer"
+	"github.com/pokt-network/poktroll/pkg/relayer/miner"
+	"github.com/pokt-network/poktroll/pkg/relayer/proxy"
+	"github.com/pokt-network/poktroll/pkg/relayer/session"
+)
+
+const omittedDefaultFlagValue = "explicitly omitting default"
+
+var (
+	flagSigningKeyName   string
+	flagSmtStorePath     string
+	flagSequencerNodeUrl string
```

Comment:
What is the difference between these flags and the cosmosflags.FlagNode?

Reply:
I updated the comments and names so PTAL